doc/rvv-intrinsic-spec.adoc

```diff
@@ -180,45 +188,48 @@ vint32m1_t __riscv_vadd_vv_i32m1_tumu(vbool32_t mask, vint32m1_t maskedoff,
 
 - Intrinsics with tail-undisturbed (`vta=0`)
 - Intrinsics with mask-undisturbed (`vma=0`)
 - Intrinsics representing Vector Multiply-Add Operations ^13^
+- Intrinsics representing Vector Slideup Instructions ^24^
 
 [[explicit-naming-scheme]]
 === Explicit (Non-overloaded) naming scheme
 
 In general, the intrinsics are encoded as the following. The intrinsics under this naming scheme are the "non-overloaded intrinsics", which in parallel we have the "overloaded intrinsics" defined under <<implicit-naming-scheme>>.
 
+The naming rules is the following.
+
 ```
 __riscv_{V_INSTRUCTION_MNEMONIC}_{OPERAND_MNEMONIC}_{RETURN_TYPE}_{ROUND_MODE}_{POLICY}{(...)
 ```
 
 * `OPERAND_MNEMONIC` are like `vv`, `vx`, `vs`, `vvm`, `vxm`
-* Depending on whether the `RETURN_TYPE` is a mask register...
-** For intrinsics that have a non-mask register as the destination register
+* `RETURN_TYPE` depends on whether the return type of the vector instruction is a mask register...
+** For intrinsics that represents instructions with a non-mask destination register:
 *** `EEW` is one of `i8 | i16 | i32 | i64 | u8 | u16 | u32 | u64 | f16 | f32 | f64`.
-*** `EMUL` is one of `m1 | m2 | m4 | m8 | mf2 | mf4 | mf8`
-** For intrinsics that have a mask register as a destination register
-*** The return type is one of `b1 | b2 | b4 | b8 | b16 | b32 | b64`, which is derived from the ratio `EEW`/`EMUL`
+*** `EMUL` is one of `m1 | m2 | m4 | m8 | mf2 | mf4 | mf8`.
+*** <<type-system>> explains the limited enumeration of EEW, LEUL pair.
+** For intrinsics that represent intrinsics with a mask destination register:
+*** `RETURN_TYPE` is one of `b1 | b2 | b4 | b8 | b16 | b32 | b64`, which is derived from the ratio `EEW`/`EMUL`.
 * `V_INSTRUCTION_MNEMONIC` are like `vadd`, `vfmacc`, `vsadd`.
-*** <<chapter-type-system>> explains the limited enumeration of EEW, LEUL pair.
-* `ROUND_MODE` is the `_rm` suffix mentioned in <<chapter-explicit-frm>>. Other intrinsics do not have this suffix
-* `POLICY` are enumerated under <<chapter-control-to-policy>>
+* `ROUND_MODE` is the `_rm` suffix mentioned in <<explicit-frm>>. Other intrinsics do not have this suffix.
+* `POLICY` are enumerated under <<policy-and-masked-naming-scheme>>.
 
-The general naming scheme is not sufficient to express intrinsics. The exceptions are enumerated under <<explicit-exception-naming>>.
+The general naming scheme is not sufficient to express intrinsics. The exceptions are enumerated in the proceeding section <<explicit-exception-naming>>.
 
 [[explicit-exception-naming]]
 === Exceptions in the explicit (non-overloaded) naming scheme
 
-This section enumerates the exceptions in the naming scheme.
+This section enumerates the exceptions in the explicit (non-overloaded) naming scheme.
 
 ==== Scalar move instructions
```

Comment:
I think for consistency the listings here should use `___riscv__vmv_x_s_...` instead of `vmv_x_s_...`.

Same for `vredsum_vs_...` below.